qiskit_experiments/framework/backend_data.py

```diff
@@ -36,7 +50,6 @@ class QiskitFakeBackendV2:
 
         pass
 
-
```

Comment:
I think this is a change since black 22 which we are still pinned to here. GitHub won't let me suggestion the fix in the comment.

Reply:
Should be fixed now